Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- live-build-20250810.orig/scripts/build/installer_debian-installer
+++ live-build-20250810/scripts/build/installer_debian-installer
@@ -982,9 +982,6 @@ EOF
 
 	if [ -n "${PATH_TO_POOL_INSIDE_LIVE_ENVIRONMENT}" ]
 	then
-		# Add to sources.list with the path that is used inside the live environment
-		# Add in the first line, to give it precedence over the online versions of the same package
-		echo "deb [trusted=yes] file:${PATH_TO_POOL_INSIDE_LIVE_ENVIRONMENT} ${LB_PARENT_DEBIAN_INSTALLER_DISTRIBUTION} ${LB_PARENT_ARCHIVE_AREAS}" > sources.list
 		cat chroot/chroot/etc/apt/sources.list >> sources.list
 		mv sources.list chroot/chroot/etc/apt/sources.list
 		VAR_LIB_APT_LISTS_FILENAME=$(echo "${PATH_TO_POOL_INSIDE_LIVE_ENVIRONMENT}/dists/${LB_PARENT_DEBIAN_INSTALLER_DISTRIBUTION}/Release" | sed 's|/|_|g')
